nextcloudappstore/core/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{% static 'assets/img/favicon.png' %}"/>
     <title>Nextcloud App Store{% block head-title %}{% endblock %}</title>
```

Comment:
I certainly see the point, but the question arises whether the ordering of "App store - Nextcloud - [Appstore subpage]" makes any sense. Perhaps in that case we should go for "[Appstore subpage] - App store - Nextcloud"? @BernhardPosselt